test/backend/certificates.spec.js

```diff
@@ -0,0 +1,88 @@
+import {assert, expect} from 'chai';
+import fs from 'fs';
+import {contains, dissoc, merge} from 'ramda';
+
+import {getSetting, saveSetting} from '../../backend/settings.js';
+import {
+    getCerts,
+    CA_HOST_URL,
+    calculateDaysToRenewal,
+    fetchCertsFromCA,
+    saveCertsLocally,
+    setRenewalJob,
+    renewCertificate
+} from '../../backend/certificates';
+import {username, accessToken} from './utils';
+
+const cleanUp = () => {
+    ['KEY_FILE', 'CERT_FILE'].forEach(fileName => {
+        try {
+            fs.unlinkSync(getSetting(fileName));
+        } catch (e) {}
+    });
+    try {
+        fs.unlinkSync(getSetting('SETTINGS_PATH'));
```

Comment:
`SETTINGS_PATH`  is a file. And you are correct, `unlinkSync` does not work for folders.